polkadot/zombienet_tests/functional/0007-dispute-freshly-finalized.toml

```diff
@@ -0,0 +1,45 @@
+[settings]
+timeout = 1000
+
+[relaychain.genesis.runtime.configuration.config]
+  max_validators_per_core = 2
+  needed_approvals = 3
+
+[relaychain]
+default_image = "{{ZOMBIENET_INTEGRATION_TEST_IMAGE}}"
+chain = "rococo-local"
+default_command = "polkadot"
+
+[relaychain.default_resources]
+limits = { memory = "4G", cpu = "2" }
+requests = { memory = "2G", cpu = "1" }
+
+  [[relaychain.node_groups]]
+  name = "honest"
+  count = 12
+  args = ["-lparachain=debug"]
+
+  [[relaychain.nodes]]
+  image = "{{MALUS_IMAGE}}"
+  name = "malus"
+  command = "malus dispute-finalized-candidates"
+  args = [ "--alice", "-lparachain=debug,MALUS=trace", "--dispute-offset=3" ]
+
+{% for id in range(2000,2004) %}
```

Comment:
also one parachain is probably fine for the test?

Reply:
Reduced it to 2 parachains. The way the malus variant is built rn it only disputes blocks it hasn't backed. I should be possible to remove this limitation if we'd like to drop it to 1 para.

Reply:
Reduced it further to 1 parachain. Even if malus will be the backer after a single rotation he will be able to resume disputing so it's not an issue.